### DISCUSSION TITLE
Running npm run dev and opening the link doesn't work

### DISCUSSION THREAD

**Discussion by brenfree** [coder/code-server#6728] [Q&A]
I'm running a code-server inside a Docker which the ui is exposed to the internet.

When I try:
```
npm run dev
> vite --host 0.0.0.0


  VITE v5.2.6  ready in 4414 ms

  ➜  Local:   http://localhost:5173/absproxy/5173/
  ➜  Network: http://172.19.0.2:5173/absproxy/5173/
  ➜  press h + enter to show help
```

It opens the port and says to navigate to http://localhost:5173, however when I open that page nothing appears. I'm using the vite with React to develop a website. I've looked through the documentation and can't establish the issue.

Anyone know how to fix this?

**Comment by bilogic** [comment]
I don't think loading `localhost` on your client's (the machine that is attached to the display you are looking at) browser, will load from vite which is running on `code-server`'s docker.

I would try `http://<code-server's IP>:8080/absproxy/5173/`, I assumed here you mapped your internet's 8080 to your docker's 5173
